.circleci/config.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           name: "Build and test"
           command: |
             if [ "$CIRCLE_NODE_INDEX" -eq 1 ]; then
-              ./gradlew testShardHibernate -i --scan --parallel --build-cache
+              ./gradlew testShardHibernate -i --scan --build-cache
```

Comment:
we can no longer run the tests in parallel because now more than one package depends on the docker containers and they don't know how to coordinate across separate jvms. Separate runs will clobber each other

Reply:
Yeah I've been thinking about this. In another project I've been running database docker containers without explicitly specified exported ports. (You export the port but don't tell docker what port number to use.) In that case docker will allocate randomly selected ports which you can look up using the API. Based on that we should be able to run many tests in parallel.

This is obviously a change that should be done outside this PR. :-) It could be something for next hack week! :-)